app/components/article-featured-video.js

```diff
@@ -179,35 +152,26 @@ export default Component.extend(RenderComponentMixin, JWPlayerMixin, {
 		});
 	},
 
-	/**
-	 * FIXME FEATURED VIDEO A/B TEST ONLY
-	 */
+	throttleOnScroll() {
+		throttle(this, this.onScrollHandler, null, 50, false);
+	},
+
 	onScrollHandler() {
 		const currentScrollPosition = window.pageYOffset,
-			requiredScrollDelimiter = this.getRequiredScrollDelimiter(),
-			hasScrollClass = this.$().hasClass(scrollClassName);
+			requiredScrollDelimiter = this.$().offset().top,
```

Comment:
kudos for not using jquery in this PR
what do you think about replacing this with
this.element.getBoundingClientRect().top?